### PR TITLE
Upgraded to use MIW 0.0.4-preview and storage v12 sdk

### PR DESCRIPTION
**Pull request by maliksahil**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: Startup.cs [hunk 1]
```diff
@@ -33,16 +33,10 @@ public void ConfigureServices(IServiceCollection services)
...
-            // Token acquisition service based on MSAL.NET
-            // and chosen token cache implementation
-            services.AddWebAppCallsProtectedWebApi(Configuration, new string[] { "user.read"}).AddInMemoryTokenCaches();
-
-            // // Add APIs
-            // services.AddGraphService(Configuration);
-            // services.AddHttpClient<IArmOperations, ArmApiOperationService>();
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                    .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
+                        .EnableTokenAcquisitionToCallDownstreamApi(null)
```
Comment (jmprieur) [inline Startup.cs:38]: If you know it already, you might want to pre-specific scopes (optional). This avoids to go though the user challenge afterwards.

**Review by jmprieur — APPROVED** [review]
LGTM.!

File: Startup.cs (hunk 1 above)
Comment (maliksahil) [reply Startup.cs:38]: Yes - the previous sample had on-demand scope, so I left it as is. If you have a strong feeling that we should ask for scopes at login, we can do that also.

Comment (jmprieur) [reply Startup.cs:38]: I think that's fine. If customers complain, we can always add the scope later.

**Pull request merged**